File: Introduccion/ejercicio 7/Carta.java
package introduccion;

public class Carta {
    private String codigo;
    private String descripcion;
    private String remitente;
    private String destinatario;

    public Carta(String codigo, String descripcion, String remitente, String destinatario) {
        this.codigo = codigo;
        this.descripcion = descripcion;
        this.remitente = remitente;
        this.destinatario = destinatario;
    }

    public String getCodigo() { 
        return codigo; }
    public String getDescripcion() { 
        return descripcion; }
    public String getRemitente() { 
        return remitente; }
    public String getDestinatario() { 
        return destinatario; }

    @Override
    public String toString() {
        return "Código: " + codigo + ", Remitente: " + remitente + ", Destinatario: " + destinatario + "\nDescripción: " + descripcion;
    }
}

File: Introduccion/ejercicio 7/carta.py
class Carta:
    def __init__(self, codigo, descripcion, remitente, destinatario):
        self.codigo = codigo
        self.descripcion = descripcion
        self.remitente = remitente
        self.destinatario = destinatario

    def get_codigo(self):
        return self.codigo

    def get_descripcion(self):
        return self.descripcion

    def get_remitente(self):
        return self.remitente

    def get_destinatario(self):
        return self.destinatario

    def __str__(self):
        return (f"Código: {self.codigo}, Remitente: {self.remitente}, "
                f"Destinatario: {self.destinatario}\nDescripción: {self.descripcion}")
